package.json

```diff
@@ -271,7 +271,6 @@
     "@babel/preset-flow": "^7.0.0",
     "@babel/preset-react": "^7.0.0",
     "@babel/register": "^7.0.0",
-    "@emotion/core": "^10.0.16",
```

Comment:
```
warning "react-storybook-addon-chapters > @emotion/styled@10.0.27" has unmet peer dependency "@emotion/core@^10.0.27"
```

Reply:
We have a number of those warnings which have been present for ages. I know at one point we got rid of all of them, but it seems that for a long time we've just been ignoring them, and it doesn't seem to cause any issues. Are you suggesting that we should look at each one and try to get rid of it?

Reply:
We could take a look at the other ones but I don't see the point of merging a patch that will add a new warning there :) Removing the other dependency should be fine, though.

Reply:
I found a couple of quick wins in terms of peer dependency warnings, so I added those in as well.